flask_monitoringdashboard/core/config/__init__.py

```diff
@@ -34,6 +34,7 @@ def __init__(self):
         self.outlier_detection_constant = 2.5
         self.sampling_period = 5 / 1000.0
         self.enable_logging = False
+        self.enable_param_logs = False
```

Comment:
Also, I think I mentioned this before, it would have been nice to separate the two functionalities:
- the sorting of requests
- the extra parameter logging
because then we could have already merged the request sorting.

Reply:
I am unable to understand why tests are failing. Can you please review and help me understand the scenario here?

Thanks.